evm-adapters/src/sputnik/forked_backend/rpc.rs

```diff
@@ -82,14 +82,13 @@ where
 
     fn block_number(&self) -> U256 {
         self.pin_block
-            .map(|block| match block {
+            .and_then(|block| match block {
```

Comment:
clippy complaint about this, I believe the master has the correctly formatted version of this